datasets/case02/queries/1002.2_coneMagColor.sql

```diff
@@ -0,0 +1,13 @@
+-- Cone-magnitude-color search
+-- See http://dev.lsstcorp.org/trac/wiki/dbQuery003
```

Comment:
This page does not exist, we should either replace this link with better one or remove it, this applies to all new files